.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,6 +3,16 @@ name: Bug report
 about: Create a report to help us improve
 ---
 
+**What part of Carbon does this bug correlate to?**
+* [ ]  All graphs (axes, legends etc)
+* [ ]  Bar graph
+* [ ]  Bubble graph
+* [ ]  Gantt chart
+* [ ]  Line graph
+* [ ]  Pie chart
+* [ ]  Scatter plot
+* [ ]  Timeline
```

Comment:
Should paired results be an item in this list?

Reply:
I forgot about that; added here: https://github.com/cerner/carbon-graphs/pull/294/commits/9086f0530a9852e5cd760dda1e6298f4a0c53231